app/utils/common/is800_2007.py

```diff
@@ -1377,14 +1415,15 @@ def cl_8_4_shear_buckling_coeff_Kv(only_at_support,c=None,d=None):
         Note:
               Reference - IS800_2007 cl.8.4.2.1 and cl.8.4.2.2
         """
-        if only_at_support == True:
+        if only_at_support:
```

Comment:
check this

Reply:
yes == True is not required. only_at_support takes boolean values